src/style.css

```diff
@@ -1,32 +1,92 @@
-.main-head {
-  margin: 3%;
+body {
+  background-color: lightgreen;
 }
 
-#section-1 {
-  gap: 5%;
+/* Basic styling for the categories container */
+#categories {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  justify-content: center;
+  gap: 2rem;
 }
 
-.secondary-head {
-  margin: 10%;
+/* Basic styling for each category */
+.category {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  max-width: 600px;
+  margin: 1rem;
+  padding: 1rem;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
 }
```

Comment:
I think it would be better to add some border-radius styles and some animations to food cards will get a better user experience.

![8](https://user-images.githubusercontent.com/85481456/232786539-093e1e2d-970b-4ec4-bf89-7ebd323c276b.png)